PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -401,7 +401,6 @@
 )
 ### Era dependent customization
 run2_miniAOD_80XLegacy.toModify( bjetNN,outputFormulas = cms.vstring(["at(0)*0.31628304719924927+1.0454729795455933","0.5*(at(2)-at(1))*0.31628304719924927"]))
-run2_miniAOD_80XLegacy.toModify( fatJetTable.variables, msoftdrop_chs = Var("userFloat('ak8PFJetsCHSSoftDropMass')",float, doc="Legacy uncorrected soft drop mass with CHS",precision=10))
```

Comment:
this is only run on old MINIAOD files (the Summer16 80X production) to fill a variable, why do you propose removing it?

Reply:
@ahinzmann

Reply:
changed to what it was before. the diff shows it one line below somehow.